packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -49,24 +55,51 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
     @state()
     private _isModalOpen = false;
 
+    @state()
+    private locale: CookieBannerLocale = defaultLocale;
+
     @property({ type: Boolean })
     public hasPrimaryActionsOnly = defaultProps.hasPrimaryActionsOnly;
 
     @property({ type: Object })
     public defaultPreferences = defaultProps.defaultPreferences;
 
-    @property({ type: Object })
-    public locale:CookieBannerLocale = defaultProps.locale;
-
     @property({ type: String })
     public cookieStatementLink = defaultProps.cookieStatementLink;
 
     @property({ type: String })
     public cookieTechnologiesLink = defaultProps.cookieTechnologiesLink;
 
+    @property({ type: Tenant })
+    public [TENANT_PROPERTY] = defaultProps.tenant;
```

Comment:
I personally think it's fine to use these as constants everywhere else in this file, but not on the property declaration line. It's entirely style preference but it isn't a pattern we're keen on in the repo. Please can you use the value here instead of the variable as the property key?
```suggestion
    public tenant = defaultProps.tenant;
```